internal/view/details.go

```diff
@@ -294,8 +295,12 @@ func (d *Details) saveCmd(evt *tcell.EventKey) *tcell.EventKey {
 
 func (d *Details) cpCmd(evt *tcell.EventKey) *tcell.EventKey {
 	d.app.Flash().Info("Content copied to clipboard...")
-	if err := clipboard.WriteAll(d.text.GetText(true)); err != nil {
-		d.app.Flash().Err(err)
+	if err := clipboard.Init(); err != nil {
+		panic(err)
```

Comment:
please return `t.app.Flash().Err(err)` message here (and all other files), use `panic` is bad practice. thanks!